src/renderers/webgl/WebGLTextures.js

```diff
@@ -1701,48 +1729,60 @@ function WebGLTextures( _gl, extensions, state, properties, capabilities, utils,
 
 		if ( ( isWebGL2 && renderTarget.samples > 0 ) && useMultisampledRTT( renderTarget ) === false ) {
 
+			const textures = renderTarget.isWebGLMultipleRenderTargets ? renderTarget.texture : [ renderTarget.texture ];
 			const width = renderTarget.width;
 			const height = renderTarget.height;
 			let mask = _gl.COLOR_BUFFER_BIT;
 			const invalidationArray = [ _gl.COLOR_ATTACHMENT0 ];
 			const depthStyle = renderTarget.stencilBuffer ? _gl.DEPTH_STENCIL_ATTACHMENT : _gl.DEPTH_ATTACHMENT;
+			const renderTargetProperties = properties.get( renderTarget );
+
+			for ( let i = 0; i < textures.length; i ++ ) {
 
-			if ( renderTarget.depthBuffer ) {
+				if ( renderTarget.depthBuffer ) {
 
-				invalidationArray.push( depthStyle );
+					invalidationArray.push( depthStyle );
 
-			}
+				}
 
-			const renderTargetProperties = properties.get( renderTarget );
-			const ignoreDepthValues = ( renderTargetProperties.__ignoreDepthValues !== undefined ) ? renderTargetProperties.__ignoreDepthValues : false;
+				const ignoreDepthValues = ( renderTargetProperties.__ignoreDepthValues !== undefined ) ? renderTargetProperties.__ignoreDepthValues : false;
 
-			if ( ignoreDepthValues === false ) {
+				if ( ignoreDepthValues === false ) {
 
-				if ( renderTarget.depthBuffer ) mask |= _gl.DEPTH_BUFFER_BIT;
-				if ( renderTarget.stencilBuffer ) mask |= _gl.STENCIL_BUFFER_BIT;
+					if ( renderTarget.depthBuffer ) mask |= _gl.DEPTH_BUFFER_BIT;
+					if ( renderTarget.stencilBuffer ) mask |= _gl.STENCIL_BUFFER_BIT;
 
-			}
+				}
 
-			state.bindFramebuffer( _gl.READ_FRAMEBUFFER, renderTargetProperties.__webglMultisampledFramebuffer );
-			state.bindFramebuffer( _gl.DRAW_FRAMEBUFFER, renderTargetProperties.__webglFramebuffer );
+				state.bindFramebuffer( _gl.READ_FRAMEBUFFER, renderTargetProperties.__webglMultisampledTempRead );
+				state.bindFramebuffer( _gl.DRAW_FRAMEBUFFER, renderTargetProperties.__webglMultisampledTempWrite );
```

Comment:
Can you please explain the purpose of `__webglMultisampledTempRead` and `__webglMultisampledTempWrite`? Why is it not possible to use the previous approach?

Reply:
`blitFramebuffer` cannot copy multiple color attachments (only the first), so I created new FBOs to copy them individually in my original demo (https://github.com/mrdoob/three.js/issues/23300#issuecomment-1100666026).

Temporary FBOs aren't needed here since three already has multi-sampled/regular FBOs handy, but we'd need to rebind their respective attachments after the blit.